src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dao/trust/BeneficialOwnerType.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.dao.trust;
+
+public enum BeneficialOwnerType {
+    CORPORATE_INTERESTED_PERSON("Corporate Interested Person"),
+    CORPORATE_GRANTOR("Corporate Grantor"),
+    CORPORATE_SETTLER("Corporate Settler"),
+    CORPORATE_BENEFICIARY("Corporate Beneficiary"),
+    INDIVIDUAL_INTERESTED_PERSON("Individual Interested Person"),
+    INDIVIDUAL_SETTLER("Individual Settler"),
+    INDIVIDUAL_GRANTOR("Individual Grantor"),
+    INDIVIDUAL_BENEFICIARY("Individual Beneficiary");
+
+    private final String beneficialOwnerType;
+
+    BeneficialOwnerType(String beneficialOwnerType) {
+        this.beneficialOwnerType = beneficialOwnerType;
+    }
+
+    public static BeneficialOwnerType findByBeneficialOwnerTypeString(String beneficialOwnerType) {
```

Comment:
the findByBeneficialOwnerTypeString method can be remove, it doesn't look like it is called by anything

Reply:
Yes, we have used this method now - thank you 👍